index.html

```diff
@@ -5,9 +5,62 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/c6f4ddbd48.js"
+      crossorigin="anonymous"
+    ></script>
     <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>My portfolio</h1>
+    <main>
+      <section class="landing">
+        <nav class="nav_bar">
+          <div class="toolbar">
+            <h1 class="welcome">Welcome</h1>
+            <ul>
+              <li><a href="">Portfolio</a></li>
+              <li><a href="">Contact</a></li>
+              <li><a href="">About</a></li>
+            </ul>
+            <span class="menu_icon"><i class="fa-solid fa-bars"></i></span>
+          </div>
+        </nav>
+        <section class="headline_container">
+          <header>
+            <div class="header">
+              <h1 class="header_name">Hey there. I’m Ezekiel</h1>
+              <h1 class="header_job">I’m a software developer</h1>
+            </div>
+            <p class="header_supporting_text">
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don’t hestiate to contact me.
+            </p>
+
+            <div class="socials">
+              <span class="icons icons1"
+                ><i class="fa-brands fa-twitter"></i
+              ></span>
+              <span class="icons icons2"
+                ><i class="fa-brands fa-linkedin-in"></i
+              ></span>
+              <span class="icons icons3"
+                ><i class="fa-sharp fa-solid fa-m"></i
+              ></span>
+              <span class="icons icons4"
+                ><i class="fa-brands fa-github"></i
+              ></span>
+              <span class="icons icons5"
+                ><i class="fa-brands fa-angellist"></i
+              ></span>
+            </div>
```

Comment:
- [ ] Please make sure, The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
ex: 
```
<ul> 
  <li> <a href='social network link'> icon </a> </li>
</ul>
```